source/_integrations/google_travel_time.markdown

```diff
@@ -20,104 +21,21 @@ A quota can be set against the API to avoid exceeding the free credit amount. Se
 
 ## Configuration
 
-To enable the sensor, add the following lines to your `configuration.yaml` file:
+To set up a `google_travel_time` sensor, go to the Integrations section in the Configuration tab of the Home Assistant UI, click the `+` button in the bottom right of the page, and search for `Google Travel Time`.
```

Comment:
This section (including the header), need to be replaced by:

`{% include integrations/config_flow.md %}`